gc.c

```diff
@@ -9398,6 +9398,10 @@ rb_memerror(void)
     rb_objspace_t *objspace = rb_objspace_of(rb_ec_vm_ptr(ec));
     VALUE exc;
 
+    // Print out pid, sleep, so you can attach debugger to see what went wrong:
```

Comment:
remove?

Reply:
I found this quite useful, so I wanted to leave it there for future reference, but we can remove it.

Reply:
@mame what do you think, can we keep it or remove it? I'm on the fence. Sometimes this kind of comment is useful for the future.

Reply:
I will use `if (0)`.